ensemble-dss/src/main/java/hec/dss/ensemble/DssDatabase.java

```diff
@@ -91,7 +98,7 @@ private static String buildDssPath(Ensemble e,int memberNumber, RecordIdentifier
 
     private String buildTimeSeriesStatPathName(RecordIdentifier timeSeriesIdentifier, Duration interval, ZonedDateTime startDateTime, ZonedDateTime issueDate, String stat) {
         DSSPathname path = new DSSPathname();
-        path.setAPart("");
+        path.setAPart(buildTimeAndVersionInfo(startDateTime,issueDate));
```

Comment:
This feels like we are also overriding the A part.  Could buildTimeAndVersionInfo() be called buildApart() to be consistent with the buildFpart().  It looks like the A part and F part are receiving the same information.